udata/core/reuse/models.py

```diff
@@ -40,12 +41,14 @@ def hidden(self):
                     db.Q(datasets__0__exists=False) |
                     db.Q(deleted__ne=None))
 
-
+decorator = mdstrip_field('description')
+@decorator
```

Comment:
There's no error.
The decorator seems to have no effect, the func_ code is never run.